app/screens/dashboard/Dashboard.tsx

```diff
@@ -15,7 +15,7 @@ const Dashboard = () => {
   const isDarkMode = useSelector((state: RootState) => state.ui.isDarkMode);
 
   useEffect(() => {
-    eventsService.updateBrowserViewTheme({ isDarkMode });
+    eventsService.switchTheme({ isDarkMode });
```

Comment:
Why do we need it?

Reply:
`updateBrowserViewTheme` ipc communication does not exist. Had to replace it with the switchTheme since they perform same function

Reply:
remove this, this logic is redundant after making 'isDarkMode' persistent